projects/addon-commerce/components/input-card-grouped/input-card-grouped.component.ts

```diff
@@ -421,4 +426,12 @@ export class TuiInputCardGroupedComponent
         this.autofilled = autofilled;
         this.autofilledChange.emit(autofilled);
     }
+
+    private initPreventScrollSubscription() {
+        typedFromEvent(this.elementRef.nativeElement, 'scroll')
```

Comment:
Even if it was the case, it could be a hostlistener. But this is most likely can be solved with CSS.

Reply:
I find that solution in 1.x version. Probably that piece of code was lost in transition to 2.0. I'll try to find css solution or rewrite with hostlistener

Reply:
I see. Ok, I suppose there's no CSS solution then.